content/dnms/cz/02-PCBs.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Desky plošných spojů a schémata zapojení
```

Comment:
Hi @hazarizk 
the markdown-files looks good. One thing, is it possible to shorten the title in `02-PCBs.md` because it overlaps into the documentation, see screenshot? 

![CleanShot 2021-02-25 at 10 55 50](https://user-images.githubusercontent.com/44444667/109136119-13538480-7758-11eb-99ec-24a5a05c8deb.png)